queries/reports/filters.js

```diff
@@ -0,0 +1,143 @@
+/* eslint-disable prettier/prettier */
+import { sparqlEscapeDate, sparqlEscapeUri } from 'mu'; // eslint-disable-line
+import { sparqlEscapeDateLocal } from '../utils.js';
+
+export function publicationDate(params) {
+    let publicationDateRange = params.filter.publicationDate;
```

Comment:
I think it'd be good practice to add a check here to ensure `params.filter` is defined, otherwise the whole service might crash if it gets bad params.